packages/terra-i18n-plugin/src/I18nAggregatorPlugin.js

```diff
@@ -10,11 +10,12 @@ import localeData from 'react-intl/locale-data/${language.split('-')[0]}';
 addLocaleData(localeData);
 
 const messages = ${JSON.stringify(messages, null, 2)};
-
-module.exports = {
-  areTranslationsLoaded: true,
-  locale: '${language}',
-  messages: messages
+const areTranslationsLoaded = true;
+const locale = '${language}'
+export {
+  areTranslationsLoaded,
+  locale,
+  messages
```

Comment:
This may have been fixed with some of the other changes but is this shorthand syntax still causing the same errors mentioned here? 

https://github.com/cerner/terra-core/issues/878

Reply:
Its worth testing and I think this will work. This isn't object shorthand anymore... its ES6 export syntax. What you referenced is object shorthand in an object to`module.exports`

Reply:
Ah, I missed that. That makes sense.

Reply:
After the code genrates, I can modify it and test in browsers. Here is the code that worked in chrome and IE10:
```javascript
export default {
  areTranslationsLoaded: true,
  locale: 'en',
  messages: messages
};
```
IE 10 did not like the short hand on the 'messages' property. Both browsers did not like it when I left out the 'default' keyword.

Reply:
I'll retract my previous comment. I had a copy/paste error. The code works good. looks like there is missing semi-colon for the declaration of 'locale'.

Reply:
Tested in this change in the Healthet@Cerner site which is using create-react-app and the build compiled successfully. So no regression on #878.